packages/Algernon.package/ALGActionVerbFilter.class/instance/specialSelection.given..st

```diff
@@ -0,0 +1,6 @@
+accessing
+specialSelection: anItemCollection given: anActionVerb
+	(anActionVerb = '=' or: [anActionVerb = 'do'])
+		ifTrue: [ ^ anItemCollection select: [ :each | each algerType class = ALGExpressionType] ].
```

Comment:
I know, this behaviour origins in the legacy code, but I think we should try to move this special casing to the ExpressionItem. E.g. the Existing "matchesAction"-Method. It would make the code cleaner and easier to implement more "special selections" in the Future.

Reply:
The thing is that `matchesAction` and the scoring are two subsequent filters. That means every item is matched against both filters.

That means special casing overwrites the whole fuzzy matching for performance reasons. I don't see how that decision could be made at the item level.

Reply:
We have moved/refactored this special casing into the item level. An item can set `isRelevantForAllQueries`.